packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -87,17 +92,28 @@ export default function ProfileMenu({
         },
         rightEmoji: hypeCampaign && '👕',
       },
-      {
-        title: 'Logout',
+    ];
+
+    if (seoSidebar === SeoSidebarExperiment.V1) {
```

Comment:
On the ticket it also mentions Pause new tab, not sure what the final call was?

Reply:
Final call was to add it to the profile top menu just like in design, can try to find the thread.

Reply:
decision here https://dailydotdev.slack.com/archives/C06S6HEUZB9/p1718019097951269?thread_ts=1718017443.993599&cid=C06S6HEUZB9

Reply:
All of the changes related to Dnd is here: https://github.com/dailydotdev/apps/pull/3215/commits/4824aee9e994ed3a2e3ae4a7285c47ef739582fb